test/Concurrency/Runtime/async_taskgroup_cancel_from_inside_child.swift

```diff
@@ -22,13 +22,13 @@ func test_taskGroup_cancel_from_inside_child() async {
   }
 
   let result = await withTaskGroup(of: Int.self, returning: Int.self) { group in
-    let firstAdded = group.spawnUnlessCancelled { [group] in // must explicitly capture, as the task executes concurrently
-      group.cancelAll() // allowed
+    let firstAdded = group.spawnUnlessCancelled {
       print("first")
       return 1
     }
     print("firstAdded: \(firstAdded)") // CHECK: firstAdded: true
 
+    group.cancelAll()
```

Comment:
rather than change it like this, it seems we should remove the entire test since the "cance_from_inside_child" is not a supported thing after all. right?

Reply:
I figured that it was still useful to test that we didn't spawn the 2nd task when the group was cancelled. It seemed like that's what the test as a whole was trying to do to begin with? Unless we have coverage for that somewhere else - I didn't check.

Reply:
> Unless we have coverage for that somewhere else - I didn't check.

It's covered by `async_taskgroup_cancel_then_spawn.swift`

We should not keep around a test called "cancel from inside child" if we don't support that, it'll cause confusion :)